src/integration-test/resources/ReadDetails/__files/test1_readDetailsByPartyIdAndEngagementOrgId/request.json

```diff
@@ -0,0 +1,6 @@
+{
+  "partyId": [
```

Comment:
Use the same indentation as the rest of the json-files?

(comment applicable on other json files as well)

Reply:
It was a mixed bag when I came here, but I'll fix.